### ISSUE TITLE
Increasing read timeout of waiting for a response to a sent message

### ISSUE THREAD

**Issue by tmberthold** [International-Data-Spaces-Association/IDS-ConfigurationManager#68]
In a demonstration case, when the Connector is registered at a Broker via the ConfigManager, the time span of the _ConfigManager -> DSC -> Broker -> DSC -> ConfigManager_ message flow may take longer than usual. In this case, it may take longer between sending a message and receiving the confirmation, which may cause a read timeout while waiting for the response from the Connector (if we would get one). 

For this, it would have to be possible to increase the time span until the exception is thrown and we assume that we will no longer receive an answer at all.

**Comment by tmberthold** [comment]
```
12:19:57.941 [http-nio-8081-exec-8] INFO - ---- [DataspaceConnectorClient updateAtBroker] updating connector localhost at broker https://broker.ids.isst.fraunhofer.de/infrastructure
12:20:07.944 [http-nio-8081-exec-8] ERROR - Read timed out
```

**Comment by tmberthold** [comment]
Answer received by connector at: 12:20:11.024 (~4 sec after timeout)

**Comment by goekhanKahriman** [comment]
fixed in "bugfix" branch with commit-id: 6547456fdfa7bda561f87713f500291d409211c2.
Timeout for reading, connecting and writing is increased to 30 seconds.
